chainer/functions/array/pad.py

```diff
@@ -0,0 +1,65 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Pad(function.Function):
+
+    """Padding of an array"""
+
+    def __init__(self, pad_width, mode, **keywords):
+        self.mode = mode
+        self.keywords = keywords
+        self.pad_width = pad_width
+        self.pad_bw = numpy.asarray(pad_width)
+        if self.pad_bw.size == 1:
+            self.pad_bw = numpy.repeat(self.pad_bw, 2)
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
```

Comment:
Could you add a comment that we can check the validity of the input in `xp.pad` (I understand so. Correct me if it is wrong).